loleaflet/css/jquery-ui-lightness.css

```diff
@@ -881,7 +881,7 @@ button.ui-button::-moz-focus-inner {
 	font-size: 1.1em;
```

Comment:
property of the intended element we are trying to target, maybe change it to 1em?